src/vaadin-grid-sort-mixin.html

```diff
@@ -22,13 +22,15 @@
           value: false
         },
 
+        /** @private */
```

Comment:
Seems like this is used in `ArrayDataProviderMixin` so should be `@protected`?

Reply:
Done.